### PR TITLE
Editor style tuning and add Youtube block

### PR DESCRIPTION
**Pull request by nickhsine**
add ```clearfix``` to fix float side-effect when aligning.

@hcchien

### CODE REVIEW

File: fields/types/html/editor/draft-converter.js [hunk 1]
```diff
@@ -36,6 +36,7 @@ let defaultEntityTagMap = {
 	image: ['<img src="<%= url %>">', '</img>'],
 	slideshow: ['<!-- slideshow component --> <ol> <% _.forEach(this, function(image) { %><li><img src="<%- image.url %>" /></li><% }); %>', '</ol>'],
 	imageDiff: ['<!-- imageDiff component --> <ol> <% _.forEach(this, function(image, index) { if (index > 1) { return; } %><li><img src="<%- image.url %>" /></li><% }); %>', '</ol>'],
+	youtube: ['<iframe width="560" height="315" src="https://www.youtube.com/embed/<% youtubeId %>" frameborder="0" allowfullscreen>', '</iframe>'],
```
Comment (hcchien) [inline fields/types/html/editor/draft-converter.js]: so this will be the fixed size iframe?

Comment (nickhsine) [inline fields/types/html/editor/draft-converter.js]: No, this is just for RSS feed if needed in the future

**Comment by hcchien** [comment]
👍

File: fields/types/html/editor/base/atomic-block-switcher.js [hunk 2]
```diff
@@ -65,34 +68,101 @@ class AtomicBlockSwitcher extends React.Component {
 			</div>
 		);
 
-		const { alignment, device } = this.props;
-		let className = alignment;
+		const device = _.get(this.props, ['blockProps', 'device'], 'mobile');
 		let BlockComponent;
+		let style;
+
+		let mobileStyle = {
```
Comment (garfieldduck) [inline fields/types/html/editor/base/atomic-block-switcher.js]: change them as constants / take them into a config (?)

**Comment by garfieldduck** [comment]
LGTM :+1:

**Pull request merged**